sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc

```diff
@@ -1083,6 +1094,36 @@ When providing the above types as `DataT` parameters to an image read function,
 the corresponding `HintT` parameters to use would be `sycl::vec<float, 4>` and 
 `sycl::vec<short, 2>`, respectively.
 
+=== Querying sampled image fetch support [[querying_sampled_image_fetch_support]]
+
+We provide the following device queries to query support for sampled image 
+fetch with various backing memory types and dimensionalities.
+
+The device aspect descriptors for these queries are:
+
+[frame="none",options="header"]
+|======================
+|Device descriptor | Description
+|`aspect::ext_oneapi_bindless_sampled_image_fetch_1d_usm` | 
+ Indicates if the device is capable of fetching USM backed 1D 
+ sampled image data.
+|`aspect::ext_oneapi_bindless_sampled_image_fetch_1d` | 
+ Indicates if the device is capable of fetching non-USM backed 1D 
+ sampled image data.
+|`aspect::ext_oneapi_bindless_sampled_image_fetch_2d_usm` | 
+ Indicates if the device is capable of fetching USM backed 2D 
+ sampled image data.
+|`aspect::ext_oneapi_bindless_sampled_image_fetch_2d` | 
+ Indicates if the device is capable of fetching non-USM backed 2D 
+ sampled image data.
+|`aspect::ext_oneapi_bindless_sampled_image_fetch_3d_usm` | 
+ Indicates if the device is capable of fetching USM backed 3D 
+ sampled image data.
+|`aspect::ext_oneapi_bindless_sampled_image_fetch_3d` | 
+ Indicates if the device is capable of fetching non-USM backed 3D 
+ sampled image data.
```

Comment:
3 more aspects and this extension will have more aspects than the core spec! I think we should start considering if we really need all those aspects or if we can mandate some of these be emulated if the device doesn't support it natively? Aspects are good, but if the user has to check 5+ aspects when they use bindless images they might get a little grumpy.

Not a blocker for this PR though.

Reply:
I agree that we may end up with more aspects than is ideal. However, I'm not sure how we can emulate any of our current aspects. A lot of them indicate whether some operation is possible on a specific device/backend, emulating the operations that our aspects entail would be a challenge, and I'm not sure if it's possible.

It is something to look into, however. Thanks for pointing it out!